tests/mock_data.py

```diff
@@ -72,3 +72,9 @@
         "service-info"
     )
 }
+MOCK_SERVICE = {
```

Comment:
Included required properties from [ga4gh schema](https://raw.githubusercontent.com/ga4gh-discovery/ga4gh-service-info/v1.0.0/service-info.yaml#/components/schemas/Service), analogous to trs_filer I left out `id` and am setting it during the test.

Reply:
Or actually you _could_ add `id`, since you're not using it in the other test anyway